spec/unit/type/filesystem_spec.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
@hunner @bmjen  So this was the problematic line (it was #!/usr/bin/env rspec). Should we leave it as is, or revert to what's on master which is nothing?